lib/rom/processor/transproc.rb

```diff
@@ -99,6 +99,8 @@ def visit(attribute)
       def visit_attribute(attribute)
         if attribute.typed?
           t(:map_value, attribute.name, t(:"to_#{attribute.type}"))
+        elsif attribute.meta[:coercer]
+          t(:map_value, attribute.name, attribute.meta[:coercer])
         end
```

Comment:
Nah man, error should be raised in `AttributeDSL#attribute` like I mentioned (: Here it's too low-level and too defensive I think.